processes/docs-contributing.md

```diff
@@ -30,7 +30,7 @@ The sort of contributions that we are looking for are:
 
 For reporting issues about a particular project, please use the relevant repository's issue tracker or post on the [Devel mailing list][mailing-list].
 
-[mailing-list]: https://sel4.systems/lists/listinfo/
+[mailing-list]: https://lists.sel4.systems/postorius/lists/
```

Comment:
This is a bit out of the scope of this PR, but I wonder if we should point people directly to the `devel` list here? Currently, the other list available are `RFC` and `announce`, which are a bit more special. Seem we will not have more list soon as the traffic is still low.

```suggestion
[mailing-list]: https://lists.sel4.systems/postorius/lists/devel.sel4.systems/
```

Aside from this, we could also recommend the Discourse forums at https://sel4.discourse.group/...

Reply:
Yes, the sentence specifically mentions the Devel list, so updating the link to that would make sense. Let's do that separately.